autoconfiguration/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+  id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+}
+
+dependencyManagement {
+  imports {
+    mavenBom "org.springframework.boot:spring-boot-dependencies:${project.ext['spring-boot.version']}"
+  }
+}
+
+archivesBaseName = 'upmc-enterprises-graceful-shutdown-spring-boot-autoconfiguration'
+
+ext['lombok.version'] = '1.18.16'
```

Comment:
Can the version of lombok use the one specified in Spring Boot's dependency management (which should be `org.projectlombok:lombok:1.18.20` for [Spring Boot 2.5.2](https://docs.spring.io/spring-boot/docs/2.5.2/reference/html/dependency-versions.html#dependency-versions.properties)) or does this version need to target `1.18.16` specifically?